packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -346,9 +346,6 @@ export class TokenRatesController extends StaticIntervalPollingController<TokenR
         );
 
         if (this.#chainId !== chainId || this.#ticker !== ticker) {
-          this.update((state) => {
-            state.marketData = {};
-          });
           this.#chainId = chainId;
```

Comment:
When we move to the new polling API (`startPolling` from the base class), this entire subscription and concept of r 'current chain id' will go away.  The caller will become responsible for passing in the chain id(s) to poll, rather than the controller listening to the current one.